matrix-editor/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "matrix-editor",
+  "version": "1.0.0",
+  "main": "lib/script.js",
+  "repository": "https://github.com/chchch/sanskrit-alignment",
+  "author": "Charles Li",
+  "license": "BSD-3-Clause",
+  "type": "module",
+  "private": true,
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "build": "webpack",
```

Comment:
Not working yet, the code would have to be refactored, then something like this for a `webpack.config.js` could work I think

```js
const path = require('path')

module.exports = {
  mode: "development",
  devtool: "inline-source-map",
  entry: {
    main: "./lib/script.js",
  },
  output: {
    path: path.resolve(__dirname, './dist'),
    filename: "index.js"
  },
  resolve: {
    extensions: [".js"],
    modules: [
      path.resolve('node_modules')
    ]
  },
}
```